hazelcast/serialization/data.py

```diff
@@ -76,17 +77,6 @@ def is_portable(self):
         """
         return CONSTANT_TYPE_PORTABLE == self.get_type()
 
-    def has_partition_hash(self):
-        """Determines whether this ``Data`` has partition hash or not.
-        
-        Returns:
-            bool: ``True`` if ``Data`` has partition hash, ``False`` otherwise.
-
-        """
-        return self._buffer is not None \
-               and len(self._buffer) >= HEAP_DATA_OVERHEAD \
```

Comment:
nit: it seems that we don't have this check anymore. Previous implementation was returning `self.hash_code()` when `len(self._buffer) < HEAP_DATA_OVERHEAD`. But I don't think this situation can be faced in the current implementation. WDYT?

Reply:
Yes, I checked that part. We try to get partition hash only when the Data is created from the ObjectDataOutput, which will always be bigger than that